chart/templates/replicated-secret.yaml

```diff
@@ -29,6 +29,9 @@ stringData:
     statusInformers:
       {{- .Values.statusInformers | toYaml | nindent 6 }}
     {{- end }}
+    userAgent: {{ .Values.userAgent | default "" | quote }}
```

Comment:
Thought: what do you think of passing this as an env var and then updating the `GetUserAgent` in the `buildversion` package to return it if set, otherwise behave as is? That would simplify the code a bit. We already pass some non-sensitive information as env vars for similar reasons. Not sure if it's an anti-pattern though.

Reply:
was not a fan of passing the userAgent around everywhere, so this works for me

Reply:
It would also limit user-error in the future if someone forgets to pass it. I'm in favor.